core/templates/domain/creator_dashboard/creator-dashboard-backend-api.service.ts

```diff
@@ -41,8 +41,7 @@ import {
 import {
   SuggestionBackendDict,
   Suggestion,
-  SuggestionObjectFactory
-} from 'domain/suggestion/SuggestionObjectFactory';
+} from 'domain/suggestion/Suggestion.model';
```

Comment:
Where is the domain/suggestion/Suggestion.model file? Did you forgot to push it?

Reply:
yeah forgot to push it , now done.